services/gundeck/test/unit/ThreadBudget.hs

```diff
@@ -149,42 +118,6 @@ tests =
       testProperty "qc stm (sequential)" propSequential
     ]
 
-----------------------------------------------------------------------
--- deterministic unit test
-
-_testThreadBudgets :: Assertion
-_testThreadBudgets = do
```

Comment:
if this passes we should probably enable it?  i think it's just disabled because it takes time...